web/satellite/src/components/account/billing/paymentMethods/AddTokenCard.vue

```diff
@@ -0,0 +1,227 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="token">
+        <div class="token__large-icon-container">
+            <div class="token__large-icon">
+                <StorjLarge />
+            </div>
+        </div>
+        <div
+            class="token__add-funds"
+        >
+            <h3
+                class="token__add-funds__title"
+            >
+                STORJ Token
+            </h3>
+            <p class="token__add-funds__label">Deposit STORJ Tokens via Coin Payments:</p>
+            <TokenDepositSelection2
+                class="token__add-funds__dropdown"
+                :payment-options="paymentOptions"
+                @onChangeTokenValue="onChangeTokenValue"
+            />
+            <div class="token__add-funds__button-container">
+                <VButton
+                    class="token__add-funds__button"
+                    label="Continue to CoinPayments"
+                    width="150px"
+                    height="30px"
+                    font-size="11px"
+                    :on-press="onConfirmAddSTORJ"
+                />
+                <VButton
+                    v-if="totalCount > 0"
+                    class="token__add-funds__button"
+                    label="Back"
+                    is-transparent="true"
+                    width="50px"
+                    height="30px"
+                    font-size="11px"
+                    :on-press="toggleShowAddFunds"
+                />
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+import StorjLarge from '@/../static/images/billing/storj-icon-large.svg';
+import VButton from '@/components/common/VButton.vue';
+import TokenDepositSelection2 from '@/components/account/billing/paymentMethods/TokenDepositSelection2.vue';
+
+import { PAYMENTS_ACTIONS } from '@/store/modules/payments';
+import { PaymentAmountOption } from '@/types/payments';
+
+const {
+    MAKE_TOKEN_DEPOSIT,
+    GET_PAYMENTS_HISTORY,
+} = PAYMENTS_ACTIONS;
+
+// @vue/component
+@Component({
+    components: {
+        StorjLarge,
+        VButton,
+        TokenDepositSelection2,
+    },
+})
+export default class AddTokenCard extends Vue {
+    @Prop({default: 0})
+    private readonly totalCount: number;
+    @Prop({default: false})
+    private readonly DEFAULT_TOKEN_DEPOSIT_VALUE = 10; // in dollars.
```

Comment:
nit - these two constants could be added to src/utils/constants - I don't mind having them defined in the component, but I do think the constants directory is well suited to this purpose